src/styles.css

```
@@ -708,3 +708,44 @@ footer a:hover{text-decoration:underline}
+.match-bar-points{font-size:10px;font-weight:700;color:var(--muted);white-space:nowrap}
+.match-bar{height:8px;border-radius:100px;background:var(--border2);overflow:hidden}
+.match-bar-fill{height:100%;width:0;border-radius:100px;background:linear-gradient(90deg,var(--orange),var(--green));transition:width .6s cubic-bezier(.4,0,.2,1)}
+.match-breakdown-panel:not(.hidden) .match-bar-fill{width:var(--match-pct,0%)}
+.match-pill-groups{display:flex;flex-direction:column;gap:8px;margin-top:12px;padding-top:10px;border-top:1px dashed var(--orange-border)}
+.match-pill-group{display:flex;flex-direction:column;gap:6px}
+.match-pill-heading{font-size:9px;font-weight:800;letter-spacing:.08em;text-transform:uppercase;color:var(--muted)}
+.match-pill-list{display:flex;flex-wrap:wrap;gap:6px}
+.match-pill{display:inline-flex;align-items:center;padding:3px 10px;border-radius:100px;font-size:10px;font-weight:700;border:1px solid transparent}
+.match-pill-lang{background:var(--white);border-color:var(--green);color:var(--green)}
+.match-pill-domain{background:var(--white);border-color:var(--blue);color:var(--blue)}
+.match-pill-lang::before,.match-pill-domain::before{content:'';display:inline-block;width:6px;height:6px;border-radius:50%;margin-right:6px;background:currentColor}
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)